corehq/apps/data_dictionary/views.py

```diff
@@ -33,12 +33,13 @@
 from corehq.apps.data_dictionary.util import (
     delete_case_property,
     get_data_dict_props_by_case_type,
-    get_used_props_by_case_type,
```

Comment:
Isn't this still used in the file, in `data_dictionary_json`?

Reply:
I believe [this answer](https://github.com/dimagi/commcare-hq/pull/36297#discussion_r2069416469) applies here as well. This branch was based on top of a branch that removed that function altogether.